src/frontend/EdFi.DataManagementService.Frontend.AspNetCore/Program.cs

```diff
@@ -34,6 +35,9 @@
 
 app.MapRouteEndpoints();
 
+app.MapHealthChecks("/health");
+app.UseHealthChecks("/health", new HealthCheckOptions { Predicate = _ => true });
```

Comment:
I don't think you need the `HealthCheckOptions` from what I read, if predicate is null it will run all the checks, which is what we want. This will do the same thing but it's confusing to developers so I would leave it out if you can.